nyaruka/rapidpro
# Drop old export models and assets app

* Drop old export foreign keys on `Notification`
 * Drop old export models
 * Update `test_emails` command to use new model
 * Delete old export deletion data migration tests - use `pragma: no cover` in migrations
 * Move storage settings system check to `utils` app
 * Remove no longer needed assets app
